### DISCUSSION TITLE
A word of thanks to @nikito

### DISCUSSION THREAD

**Discussion by keith721** [toverainc/willow-inference-server#132] [General]
After waiting a few months, I am very pleased how Willow's look and feel is beginning to come together.  What impresses me most is the rapid and courteous support provided to me this weekend by Nick Bento (@nikito).   I had to edit the WIS ubuntu.sh script to fool it into believing my Debian 12 system is actually Debian 11.  No problem there, I was familiar with this from my first attempts with WIS weeks ago.  It turned out, the nvidia Cuda libraries consume a LOT of disk space on /usr/local/ and the / partition on my 1 TB drive was too small.  Re-installed Debian, put everything on a single partition, and boom, successful install again.   When I tried loading WAS, I followed the Github notes instead of [the Quick Start Guide](https://heywillow.io/quick-start-guide/#install-was) on the Willow web site.  @nikito to the rescue again. Do it the easy way - duh.

I do believe great things are in store for all the Willow developers and users.  It's because of you helpful folks that more people can use it sooner.  I personally have a good bit of learning to do, but so VERY glad this community is helpful.  

Meanwhile thanks again.

**Comment by nikito** [comment]
Thank you for the kind words, I'm glad I could help! I hope you enjoy Willow and have fun learning all the awesome things that you can do with it!
